Assets/Scripts/UI/GameOverMenu/GameOver.cs

```diff
@@ -7,11 +7,11 @@ public class GameOver : MonoBehaviour
 {
    public void Restart()
     {
-        SceneManager.LoadScene("TrackTest");
+        SceneManager.LoadScene(2);
```

Comment:
That's better but to make the code more robust and more reusable you should use SceneManager.GetActiveScene().buildIndex instead of hardcoding 2 in there